config/deprecation-workflow.js

```diff
@@ -5,7 +5,7 @@ self.deprecationWorkflow.config = {
         { handler: 'silence', matchId: 'ember-inflector.globals' },
         { handler: 'silence', matchId: 'ember-metal.get-with-default' },
         { handler: 'silence', matchId: 'computed-property.volatile' },
-        { handler: 'silence', matchId: 'deprecate-router-events' },
+        { handler: 'throw', matchId: 'deprecate-router-events' },
```

Comment:
I've been just removing these lines. Will this make it throw an error instead, making it more obvious when it happens?

Reply:
Yea, this is going away once I address the test failures